mayastor/src/host/blk_device.rs

```diff
@@ -28,17 +28,43 @@ use std::{
 };
 
 use proc_mounts::{MountInfo, MountIter};
-use rpc::mayastor::{
-    block_device::{Filesystem, Partition},
-    BlockDevice,
-};
 use udev::{Device, Enumerator};
 
 // Struct representing a property value in a udev::Device struct (and possibly
 // elsewhere). It is used to provide conversions via various "From" trait
 // implementations below.
 struct Property<'a>(Option<&'a OsStr>);
 
+pub struct Partition {
```

Comment:
Could you please clarify why we have local copies of the structures defined via '.proto' files ?
It's a bit of unusual to have a copy of auto-generated GRPC structure.

Reply:
The local copies are maintained because once the proto files are versioned, we cannot use the proto definitions in here.  So we use the local copies and covert to the proto definition struct when we want. Also we wanted to have the proto structs to be confined only to the code sections / modules which use the grpc.

eg: if we use rpc::Partition here, then for the v1 version, we will have a different struct which we have to convert to.  So this struct acts like kind of a middleware which you can convert to structs in different proto versions (v0::Partition, v1::Partition etc).

Reply:
Thanks for the explanation.
Could you please add comments to these local structures about that ?

Reply:
Done.